travis/do_riscv.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# The prebuilt toolchains only support hardfloat, so we only
+# test these for now.
+case $1 in
+  "rv32iv")
+  TARBALL=riscv32-glibc-ubuntu-20.04-nightly-2023.02.25-nightly.tar.gz
```

Comment:
Is there a reason `2023.02.25` is hard-coded? In `build-riscv.sh`, I use something like this:
```
TAG=$(curl -s https://github.com/riscv-collab/riscv-gnu-toolchain/releases/ \
                  | perl -ne 'if(/(\d\d\d\d\.\d\d\.\d\d)/){print $1; exit}')
```

Then replace all instances of `2023.02.25` with `$TAG`.

Reply:
Even if it must be fixed at `2023.02.25` for now, I would put it in a variable like `TAG` and then use `$TAG` everywhere instead of copying `2023.02.25` in several places.

Reply:
Thanks. The main reason for this tag is that it is tested.